### ISSUE TITLE
Missing accessibility props

### ISSUE THREAD

**Issue by SuhairZain** [n4kz/react-native-material-ripple#21]
Hi,
Out of the accessibility props, only `accessible` and `accessibilityLabel` are passed on to the `TouchableWithoutFeedback` component, with other accessibility-related props like `accessibilityTraits` and `accessibilityComponentType` being passed to the `View` instead. Can all the accessibility-related props be filtered to be passed to the `TouchableWithoutFeedback` instead?

Also, wondering how good of an idea it'd be to have an extra prop `accessibilityProps` on the Ripple component, with which the user can pass in any accessibility-related props and these will be passed to the `TouchableWithoutFeedback`? This way, even if new props are added, a new release of this library wouldn't have to be made to support them.

**Comment by n4kz** [comment]
Thanks for issue and sorry for delayed reply. I've released new version `0.9.0` and added `accessibilityHint` prop to `TouchableWithoutFeedback`. I could not find `accessibilityTraits` and `accessibilityComponentType` in TouchableWithoutFeedback.propTypes. Can you give any usage examples for those props? Feel free to reopen this issue or submit a PR if you find a way to further improve accessibility of this component.
